### PR TITLE
Add LaunchableClient and LAUNCHABLE_BASE_URL env

### PR DESCRIPTION
**Pull request by ninjinkun**
To became API base URL replaceable, I introduced LaunchableClient.
- Add LaunchableClient
- Add LAUNCHABLE_BASE_URL environment variable

### CODE REVIEW

File: launchable/utils/http_client.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import requests
+import os
+import platform
+from launchable.version import __version__
+
+
+BASE_URL_KEY = "LAUNCHABLE_BASE_URL"
+DEFAULT_BASE_URL = "https://api-static.mercury.launchableinc.com"
```
Comment (shuheiktgw) [inline launchable/utils/http_client.py:8]: @ninjinkun The default base URL should be `https://api.mercury.launchableinc.com`. `api-static` subdomain is used only when you want static IPs.

**Comment by shuheiktgw** [comment]
@ninjinkun I left one comment but except that it looks good to me!

File: launchable/utils/http_client.py (hunk 1 above)
Comment (ninjinkun) [reply launchable/utils/http_client.py:8]: Oh, I see

**Comment by ninjinkun** [comment]
@shuheiktgw Thank you for your review. I fixed the API host.

**Pull request merged**